common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -214,6 +214,10 @@ class ProblemBlock(
         values={"min": 0}, scope=Scope.settings
     )
     due = Date(help=_("Date that this problem is due by"), scope=Scope.settings)
+    due_num_weeks = Integer(
```

Comment:
I wonder if you need this field at all. I don't think we want to allow setting this on a per problem basis and would rather keep it at a higher level in sequentials. Having the seq_module + inheritance changes might be enough. Maybe run some quick manual tests to see if it works without these lines.